winlogbeat/etc/fields.yml

```diff
@@ -164,6 +221,11 @@ eventlog:
       description: >
         The type of account associated with this event.
 
+    - name: version
+      type: int
```

Comment:
The type in elasticsearch seems to be "integer" an not "int": https://www.elastic.co/guide/en/elasticsearch/reference/current/number.html#number Not sure if both works. We use int also in all other `fields.yml` so we would have to change it everywhere.